authors/apps/articles/renderers.py

```diff
@@ -26,3 +26,27 @@ def render(self, data, media_type=None, renderer_context=None):
                 'articles': data,
                 'articlesCount': len(Article.objects.all())
             })
+
+
+class CommentRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+        '''
+        Return a dictionary with 2 entries:
+        key = "articles", value = a LIST of articles
+        key = "articlesCount", value = number of articles
```

Comment:
Remove these comments from the Comment Renderer

Reply:
this is a documentation that informs the use what the Renderer does